### ISSUE TITLE
Possible to add some previous versions to conda-forge?

### ISSUE THREAD

**Issue by DManowitz** [conda-forge/finetuning-scheduler-feedstock#24]
### Comment:

Right now on conda-forge, there are no versions of this package between 0.2.3 and 2.0.0.  Version 0.2.3 supports pytorch-lightning up to version 1.7.7, and version 2.0.0 of this package requires at least v2.0.0 of pytorch-lightning.  However, there are quite a number of versions of pytorch-lightning between 1.7.7 and 2.0.0, and quite a few other packages do not yet support pytorch-lightning 2.0.0, so would it be possible to add some of the other versions of this package between 0.2.3 and 2.0.0 to conda-forge to enable better support for other versions of pytorch-lightning, but allow for better compatibility with other packages?

**Comment by speediedan** [comment]
Apologies for the delayed response @DManowitz! I've been consumed the last couple weeks attempting to ensure some PyTorch and Lightning PRs are in before the `2.1.0` feature window closes. 

Unfortunately, Lightning has been somewhat inconsistent with publishing conda-forge patch releases (which of course `finetuning-scheduler` depends on). The version gap you're alluding to emerged when Lightning stopped publishing `conda-forge` versions for a number of months and then published a large number of versions on one day:
 
![image](https://github.com/conda-forge/finetuning-scheduler-feedstock/assets/7462936/25e534b7-3982-4b84-9eac-ef474470b560)

I often find when I prepare a `finetuning-scheduler` release, Lightning will have a corresponding pypi release but no associated conda-forge release for some time (or at all).

It's great to know there are users interested in the `conda-forge` version of this package! As an individual maintainer with a lot on my plate, I don't have a lot of bandwidth at the moment but I will make an earnest effort to carve out some time to retroactively fill in to some degree the `0.2.3` - `2.0.0` gap you reference. I'll also certainly try to ensure moving forward that I publish conda-forge `finetuning-scheduler` releases that pair with Lightning patch releases whenever Lightning makes them available in a fairly timely fashion.

Thanks again for your feedback and understanding.

**Comment by speediedan** [comment]
@DManowitz I had a chance this morning to publish [conda-forge versions](https://anaconda.org/conda-forge/finetuning-scheduler/files) of `finetuning-scheduler` associated with the terminal patches of the last two pre-2.x Lightning minor releases (i.e. Lightning 1.8.6: FTS 0.3.4, Lightning 1.9.4: FTS 0.4.1). 
![image](https://github.com/conda-forge/finetuning-scheduler-feedstock/assets/7462936/bce303fb-4f24-4e87-8dfb-56e34c6306f6)

Thanks very much for raising this to my attention. As I indicated above, I'll ensure I issue conda-forge packages for new versions `finetuning-scheduler` moving forward whenever corresponding Lightning versions are available. 

Let me know if this is sufficient to close this issue for now from your perspective or if there are other specific patch versions that you perceive to be critical.
